File: backend/src/main/java/com/coing/infra/upbit/enums/EnumUpbitRequestType.java
package com.coing.infra.upbit.enums;

import lombok.Getter;

/**
 * Upbit WebSocket Type
 */
@Getter
public enum EnumUpbitRequestType {
	TICKER("ticker"),
	TRADE("trade"),
	ORDERBOOK("orderbook"),
	CANDLE("candle");

	private final String value;

	EnumUpbitRequestType(String value) {
		this.value = value;
	}
}

File: backend/src/main/java/com/coing/infra/upbit/enums/EnumUpbitRequestType.kt
package com.coing.infra.upbit.enums

/**
 * Upbit WebSocket Type
 */
enum class EnumUpbitRequestType(val value: String) {
    TICKER("ticker"),
    TRADE("trade"),
    ORDERBOOK("orderbook"),
    CANDLE("candle")
}
